content/asp.net/getting-started/configuring-different-environments.md

```diff
@@ -0,0 +1,153 @@
+# Configuring Different Environments
+by [Eric Fleming](https://ericflemingblog.wordpress.com)
+
+In ASP.NET Core controlling application behavior across multiple environments, such as developement, staging, and production, has been improved through the expanded use of environment variables. Environment variables are used to indicate which environment the application is running in, and can be detected programmatically allowing the application to be configured appropriately.
+
+## Environment Based Settings Files
+The constructor of the Startup class, found below, provides the ability to use more than one appsettings.json file by leveraging these environment variables. Since the `appsettings` configurations are [read in the order they are specified](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/configuration), the general `appsettings` configuration will be loaded first, followed by an environment specific `appsettings` configuration.
```

Comment:
It's not clear what impact the order in which they're loaded has. Update to make clear that the "last setting loaded" wins if a setting exists in multiple places. You can look at the docs for an example of how to phrase this, or come up with something that works for you.